source/getting-started/core-concepts.md

```diff
@@ -1,58 +1,120 @@
-To get started with Ember.js, there are a few core concepts you
-should understand.
+Before moving to more focused topics, it is important to review several
```

Comment:
"review" - this isn't a review, it's the first time they've seen these concepts.